functions/src/firestore-types/lobby.ts

```diff
@@ -69,6 +74,14 @@ export type Lobby = {
    * expiration time of the current phase with a timer
    */
   expiration?: Timestamp;
+  /**
+   * the player that was voted off for the round
+   */
+  votedOff?: string;
```

Comment:
there should be a note here about the `NONE` specail value

maybe even set the type to `string | "NONE"`